### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by proofmancer**
### Changes

- Updated the export of shared components for easier use and less cluttered code
- Finished the landing page with all of the styles and mobile responsive; also added the links for the docs ("Restake" button at the footer leads to "app.yieldi.xyz" but because we don't have anything there, it is a broken link)
- Added config for gradient circles and a wrapper component
- Updated tailwind config
- Updated button and slide menu for mobile
- Updated layout on the App so it can use the navbar properly
- Updated to PNPM

### Breaking changes:
- The problem was with the navbar; because it was sticky and we had a lot of absolute gradients on the page, the overflow didn't take effect and made the navbar go off-screen. So I added it to be fixed, but no,w when we are using it on a page or a layout, we should add a margin-top = navbar height so we can use the other styles accordingly

### CODE REVIEW

File: landing/components/common/CTA.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    link: "https://t.me/yieldi_labs",
+  },
+  {
+    icon: <Discord />,
+    link: "https://discord.com/invite/8aFPCXPwAd",
+  },
+];
+
+const CTA = () => (
+  <div className="relative flex flex-col gap-[88px] font-gt-america-exp px-8 tablet:px-18 text-neutral-900 text-center font-bold pb-[35px] tablet:pb-[115px] overflow-hidden">
```
Comment (MattNguyen) [inline landing/components/common/CTA.tsx:21]: My preference is that we do not hardcode values and instead enhance tw's default size scaling or use a value that's closest to an existing size value.

File: landing/components/common/CTA.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    <h2 className="text-[16px] tablet:text-[38px]">
```
Comment (MattNguyen) [inline landing/components/common/CTA.tsx]: text-[16px] -> text-base

File: shared/components/ui/Button.tsx [hunk 3]
```diff
@@ -5,15 +5,21 @@ interface ButtonProps {
...
-const Button = ({ label, className = "", onClick }: ButtonProps) => (
-  <button
-    onClick={onClick}
-    className={`bg-primary hover:bg-primary/50 rounded-full px-9 py-3 font-semibold text-gray-900 transition duration-300 ease-in-out ${className}`}
-  >
-    {label}
-  </button>
-);
+const Button = ({ label, className = "", onClick, link }: ButtonProps) => {
+  const baseClass = `bg-primary hover:bg-primary/50 rounded-full px-9 py-3 font-semibold text-gray-900 transition duration-300 ease-in-out ${className}`;
```
Comment (MattNguyen) [inline shared/components/ui/Button.tsx]: use twmerge

File: shared/components/ui/Button.tsx [hunk 4]
```diff
@@ -5,15 +5,21 @@ interface ButtonProps {
...
+
+  return link ? (
+    <a href={link} className={baseClass} rel="noopener noreferrer">
```
Comment (MattNguyen) [inline shared/components/ui/Button.tsx]: why not use nextjs link component?

File: shared/components/ui/Navbar/index.tsx [hunk 5]
```diff
@@ -37,44 +37,45 @@ const Navbar = ({ links, buttons }: NavbarProps) => {
...
         style={{
           transition: "transform 0.3s ease-in-out",
           transform: isSticky ? "translateY(0)" : "translateY(-10px)",
         }}
       >
-        <Link href="/">
-          <Yieldi className="tablet:w-[108px] w-[96px]" />
-        </Link>
+        <div
+          className={`bg-transparent-radial text-neutral tablet:px-10 tablet:py-8 tablet:mx-18 mx-8 flex w-full ${isSticky ? "shadow-md" : ""} items-center justify-between rounded-2xl border-4 border-white px-4 py-4 backdrop-blur-[30px]`}
```
Comment (MattNguyen) [inline shared/components/ui/Navbar/index.tsx]: twmerge?

**Review by MattNguyen — APPROVED** [review]
Looks good other than a few non-blocking comments. we should really try to avoid hardcoding tailwinds values and instead rely on the theme, sizing scale, etc. Let's merge this in so that you move on to wallet stuff and come back to clean the tailwinds stuff later.

File: shared/components/ui/Button.tsx (hunk 4 above)
Comment (proofmancer) [reply shared/components/ui/Button.tsx]: added

File: shared/components/ui/Button.tsx (hunk 3 above)
Comment (proofmancer) [reply shared/components/ui/Button.tsx]: added

**Pull request merged**